MO4/Sniffs/Commenting/PropertyCommentSniff.php

```diff
@@ -158,13 +161,18 @@ protected function processTokenWithinScope(
             $isCommentOneLiner
                 = $tokens[$commentStart]['line'] === $tokens[$commentEnd]['line'];
 
-            $length         = ($commentEnd - $commentStart + 1);
+            $length         = (int) ($commentEnd - $commentStart + 1);
             $tokensAsString = $phpcsFile->getTokensAsString(
                 $commentStart,
                 $length
             );
 
-            $varCount = (count(preg_split('/\s+@var\s+/', $tokensAsString)) - 1);
+            $vars = preg_split('/\s+@var\s+/', $tokensAsString);
+            if ($vars === false) {
+                throw new RuntimeException('This should never happen.');
```

Comment:
Can we add a slightly more useful exception message? It might be slightly frustrating if someone actually sees this :D